bucket/processhacker-nightly.json

```diff
@@ -25,6 +25,10 @@
             "Process Hacker"
         ]
     ],
+    "persist": [
+        "ProcessHacker.exe.settings.xml",
+        "usernotesdb.xml"
```

Comment:
Would this need to be created if it doesn't exist?

Reply:
It exists as an empty file in the zip file itself.